core/http-auth-aws-crt/src/main/java/software/amazon/awssdk/http/auth/aws/crt/internal/signer/DefaultAwsCrtV4aHttpSigner.java

```diff
@@ -44,9 +44,16 @@
  * An implementation of a {@link AwsV4aHttpSigner} that uses properties to compose v4a-signers in order to delegate signing of a
  * request and payload (if applicable) accordingly.
  */
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
sorry, one more point. I think this should now be moved to `software.amazon.awssdk.http.auth.aws.crt` instead of under `internal`

Reply:
Pushed a new commit to move the class and test.